apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -9,128 +10,464 @@ const svgProps: SvgIconProps = {
 };
 const iconProps = { svgSource: svgProps };
 
-const PrimaryTest: React.FunctionComponent = () => {
+const PrimaryTestWithAutoHide: React.FunctionComponent = () => {
+  const [visible, setVisible] = React.useState(false);
+  const onButtonPress = () => setVisible(!visible);
+
+  const [hidden, setHidden] = React.useState<boolean>(!visible);
+  const height = React.useRef(new Animated.Value(0)).current;
+
+  const show = () => {
+    setHidden(false);
+    Animated.sequence([
+      Animated.timing(height, {
+        toValue: -50,
+        duration: 600,
+        useNativeDriver: true,
+        easing: Easing.elastic(1.5),
+      }),
+      Animated.delay(3000),
+      Animated.timing(height, {
+        toValue: 0,
+        duration: 250,
+        useNativeDriver: true,
```

Comment:
Because I happen to know that win32 does not have a native driver, it _might_ be worth pulling this out into a separate variable should we ever decide to bring notification to other platforms and need to do a. platform check. _Might_, I'm still unsure we'll ever bring this to desktop.